tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -262,6 +381,86 @@ static void test_tx_cb_multi(void)
 	PASS();
 }
 
+static void test_tx_proxy_mixin(void)
+{
+	static struct bt_mesh_prov prov = {
+		.uuid = test_prov_uuid,
+	};
+	int err;
+
+	/* Initialize mesh stack and enable pb gatt bearer to emit beacons. */
+	bt_mesh_device_setup(&prov, &comp);
+	err = bt_mesh_prov_enable(BT_MESH_PROV_GATT);
+	ASSERT_OK(err, "Failed to enable GATT provisioner");
+
+	/* Start pb gatt beacons with 100 msec interval */
+	k_sleep(K_MSEC(1800));
+
+	LOG_INF("Provision device under test");
+	/* Provision dut and start gatt proxy beacons. */
+	bt_mesh_provision(test_net_key, 0, 0, 0, adv_cfg.addr, adv_cfg.dev_key);
+
+	/* Start proxy beacons with 1 sec interval */
+	k_sleep(K_MSEC(6000));
+
+	/* Send adv data */
```

Comment:
The comments could be a bit more meaningful. Comments rephrasing a function name aren't really helpful. E.g. here it could be something like:
```
/* Send a mesh message while advertising proxy service. Advertising the proxy service should be resumed after finishing advertising the message. */
```